src/query/storages/fuse/src/operations/replace_into/mutator/mutator_replace_into.rs

```diff
@@ -77,9 +79,27 @@ impl ReplaceIntoMutator {
                 };
                 Ok(MergeIntoOperation::Delete(delete_action))
             }
-            ColumnHash::Conflict => Err(ErrorCode::StorageOther(
-                "duplicated data detected in merge source",
-            )),
+            ColumnHash::Conflict(conflict_row_idx) => {
+                let conflict_description = {
+                    let conflicts = columns
+                        .iter()
+                        .zip(self.on_conflict_fields.iter())
+                        .map(|(col, field)| {
+                            let col_name = &field.table_field.name;
+                            // if col.index(row_idx) is None, an exception will already be thrown in build_column_hash
+                            let col_value = col.index(conflict_row_idx).unwrap().to_string();
+                            format!("\"{}\":{}", col_name, col_value)
```

Comment:
If `col_value` is large, this maybe a problem :)

Reply:
yeah that's a problem